builder_automate.md

```diff
@@ -0,0 +1,389 @@
+    touch bldr.env
+    ```
+
+    Or, if you need more explanations about the contents of the `bldr.env` file, copy the existing sample file:
+
+    ```bash
+    cp bldr.env.sample bldr.env
+    ```
+
+1. Edit `bldr.env`:
+
+      * The `APP_SSL_ENABLED` configuration must correspond with the type of hypertext transfer protocol named in `APP_URL`.
+
+        * To disable SSL, use `APP_SSL_ENABLED=false` and the `APP_URL` starts with `http`.
+        * To enable SSL, use `APP_SSL_ENABLED=true` and the `APP_URL` starts with `https`.
+
+      * Always be closing. Close the Builder addresses provided in `APP_URL` and `OAUTH_REDIRECT_URL` with a forward slash, `/`.
+
+        * `https://chef-builder.test` will NOT work.
+        * `https://chef-builder.test/` will work.
+
+
+  This `bldr.env` example shows an on-prem SSL-enabled Habitat Builder authenticating using Chef Automate's OAuth.
+  `APP_SSL_ENABLED=true` and the `APP_URL` starts with `https`.
+
+    ```bash
+    #!/bin/bash
+
+    # The endpoint, key and secret for your Minio instance (see README)
+    # Change these before the first install if needed
+    export MINIO_ENDPOINT=http://localhost:9000
+    export MINIO_BUCKET=habitat-builder-artifact-store.local
+    export MINIO_ACCESS_KEY=depot
+    export MINIO_SECRET_KEY=password
+    export APP_SSL_ENABLED=true
+    export APP_URL=https://chef-builder.test/
+
+    # The OAUTH_PROVIDER values for Chef-Automate
+    export OAUTH_PROVIDER=chef-automate
+    export OAUTH_USERINFO_URL=https://chef-automate.test/session/userinfo
+    export OAUTH_AUTHORIZE_URL=https://chef-automate.test/session/new
+    export OAUTH_TOKEN_URL=https://chef-automate.test/session/token
+    export OAUTH_SIGNUP_URL=https://github.com/join
+
+    # The OAUTH_REDIRECT_URL is the registered OAuth2 redirect
+    # IMPORTANT: If SSL is enabled, the redirect URL should be https
+    # IMPORTANT: don't forget the `/` at the end of the URL
+    export OAUTH_REDIRECT_URL=https://chef-builder.test/
+
+    # The OAUTH_CLIENT_ID is the registered OAuth2 client id
+    export OAUTH_CLIENT_ID=0123456789abcdef0123
+
+    # The OAUTH_CLIENT_SECRET is the registered OAuth2 client secret
+    export OAUTH_CLIENT_SECRET=0123456789abcdef0123456789abcdef01234567
+
+    # Modify these only if there is a specific need, otherwise leave as is
+    export BLDR_CHANNEL=on-prem-stable
+    export BLDR_ORIGIN=habitat
+    export HAB_BLDR_URL=https://bldr.habitat.sh
+
+    # Help us make Habitat better! Opt into analytics by changing the ANALYTICS_ENABLED
+    # setting below to true, then optionally provide your company name. (Analytics is
+    # disabled by default. See our privacy policy at https://www.habitat.sh/legal/privacy-policy/.)
+    export ANALYTICS_ENABLED=false
+    export ANALYTICS_COMPANY_NAME=""
+    ```
+
+### Step Three: Put the Certs with the Install Script
+
+Copy the any custom certificate `.crt` and `.key` files to the same location as the `./install.sh` script.
```

Comment:
Maybe add a note calling out that the cert files MUST be named "ssl-certificate.crt" and "ssl-certificate.key", and that they should rename as needed.